sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -88,6 +88,7 @@ export { BaseRequestPolicy }
 export interface CloseHandlesInfo {
     // (undocumented)
     closedHandlesCount: number;
+    numberOfHandlesFailedToClose?: number;
```

Comment:
`numberOfHandlesFailedToClose` is inconsistent with `closedHandlesCount` :(

Reply:
regarding naming

Reply:
Since we already shipped `closeHandlesCount` it make sense to be consistent with released API so `closeFailureCount` is better and aligning with Python.

Reply:
Nice catch. Agree, let's use closedHandlesCount.

Reply:
you meant closeFailureCount?